geomesa-blobstore/geomesa-blobstore-api/src/main/java/org/locationtech/geomesa/blob/api/Blob.java

```diff
@@ -0,0 +1,41 @@
+package org.locationtech.geomesa.blob.api;
+
+
+public class Blob {
+
+    private String id;
```

Comment:
we don't have any mechanism or functionality with mime types yet

we could add this though